birdhouse/components/finch/service-config.json.template

```diff
@@ -1,6 +1,10 @@
 {
   "$schema": "https://raw.githubusercontent.com/DACCS-Climate/Marble-node-registry/main/node_registry.schema.json#service",
```

Comment:
Let's update this to specify version 1.2.0 as well

Reply:
And then update the test in `tests/test_deployment.py` so that it gets the schema from the "$schema" keyword in the json file instead of using the `services_config_schema` pytest fixture.

That way, we can ensure that the schema is validated by the correct version of the schema each time.

Reply:
Released! 

(https://github.com/DACCS-Climate/Marble-node-registry/releases/tag/1.2.0)